CONTRIBUTING.md

```diff
@@ -107,6 +105,26 @@ make coverage
 make htmlcov
 ```
 
+#### Lint Iter8
+```shell
+make lint
+```
+
+#### Build and push Iter8 image
```

Comment:
This is great. How are you using the docker images you're building locally and pushing to your Docker hub account? If you could provide some examples (e.g., how a new image value can be supplied while launching an Iter8 experiment), that would be useful.